src/delta.rs

```diff
@@ -180,7 +188,7 @@ where
             continue;
         }
 
-        if state == State::FileMeta && should_handle(&State::FileMeta, config) {
+        if state == State::FileMeta && should_handle(&State::FileMeta, config) && !config.color_only {
```

Comment:
I think this can be refactored after `commit style` and `hunk header style` are also resolved.